OracleWebLogic/samples/12213-patch-wls-for-k8s/README.md

```diff
@@ -2,24 +2,24 @@ Example of Image with WLS Domain
 ================================
 There are two  Dockerfiles which extend the Oracle WebLogic install image and apply ia necesary patch for the WebLogic Kubernetes Operator 2.0. 
 
-	1)Dockerfile.patching-ontop-12213: applies two patches `p28076014` (on top of WLS 12.2.1.3) and `p27117282` (this patch is needed only if the WebLogic binary image is created manually from this Github project).

```

Comment:
Typos in the first sentence: install image and apply ia necesary patch -> install image and apply a necessary patch. Other than that, it looks ok to me.